<launch>
    <arg name="model"/>
    <arg name="dataset"/>
    <arg name="epochs" default="1"/>
    <arg name="validate_only" default="false"/>
    <arg name="visualize_validation" default="false"/>

    <node pkg="rr_ml" type="train_seg" name="train_seg" output="screen">
        <param name="epochs" value="$(arg epochs)"/>
        <param name="data_dir" value="$(find rr_ml)/data/segmentation/$(arg dataset)"/>
        <param name="model_path" value="$(find rr_ml)/models/segmentation/$(arg model).h5"/>
        <param name="validate_only" value="$(arg validate_only)"/>
        <param name="visualize_validation" value="$(arg visualize_validation)"/>
    </node>
</launch>
